plugins/language/ThemeValues.qml

```diff
@@ -0,0 +1,78 @@
+/*
+ * This file is part of system-settings
+ *
+ * Copyright (C) 2013 Canonical Ltd.
+ *
+ * Contact: Sebastien Bacher <sebastien.bacher@canonical.com>
+ *
+ * This program is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 3, as published
+ * by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful, but
+ * WITHOUT ANY WARRANTY; without even the implied warranties of
+ * MERCHANTABILITY, SATISFACTORY QUALITY, or FITNESS FOR A PARTICULAR
+ * PURPOSE.  See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+// THIS FILE IS LOADED DIRECTLY BY NAME FROM THE SECURITY-PRIVACY PANEL.
+// IF YOU RENAME IT YOU MUST UPDATE THE REFERENCE THERE.
```

Comment:
Hmm, I don't think this file is loaded by the security-privacy panel. You'll need to update the copyright notification to assign copyright of this file to yourself, too.

Reply:
I've updated the copyright notification. Kindly review again, not sure if I did them right 😄 
Thanks!